packages/backend/src/open_payments/auth/middleware.ts

```diff
@@ -45,13 +45,15 @@ export function createAuthMiddleware({
       if (!access) {
         ctx.throw(403, 'Insufficient Grant')
       }
-      try {
-        if (!(await verifySigAndChallenge(grant.key.jwk, ctx))) {
-          ctx.throw(401, 'Invalid signature')
+      if (!config.bypassSignatureValidation) {
```

Comment:
Log might be handy when signature validation is being bypassed, could also do an early return to avoid nesting

Reply:
>could also do an early return to avoid nesting

I think we do still want the `ctx.grant = grant` below